scripts/lxd-container/README.adoc

```diff
@@ -0,0 +1,10 @@
+= Introduction of deployment in LXD
+Min Chen <vanderchen@outlook.com>
+v0.1, 2020-09-18
+:toc: right
+:sectnums:
+:imagesdir: images
+
+== Install LXD and disrtibuilder
+
+== Build LXD container images
```

Comment:
@VanderChen Please give some paragraphs here to describe how to use those scripts in this folder.